ark/utils/segmentation_utils_test.py

```diff
@@ -46,6 +46,61 @@ def test_find_nuclear_mask_id():
         assert predicted_nuc == true_nuc_ids[idx]
 
 
+def test_split_large_nuclei():
+    cell_mask, _ = test_utils.create_test_extraction_data()
```

Comment:
thank you `test_utils`, very cool

btw, do you think any of this test function should get moved/incorporated into `test_utils`?  Could be useful to have simple but 'generalized' cell+nuclear mask data.

Reply:
If we have a second function that needs it, then I'd say worth it. Until then we can keep it here